dev/breeze/src/airflow_breeze/commands/testing_commands.py

```diff
@@ -270,7 +278,7 @@ def run_tests_in_parallel(
     memory_available = psutil.virtual_memory()
     if memory_available.available < LOW_MEMORY_CONDITION and exec_shell_params.backend in ["mssql", "mysql"]:
         # Run heavy tests sequentially
-        heavy_test_types_to_run = {"Core", "Providers"} & set(test_types_list)
+        heavy_test_types_to_run = {"Core"} & set(test_types_list)
```

Comment:
I **think** this will make Providers tests safe to run for Public runners - each of them will require much less of memory and the way it is done where each "provider" test will run in their separate docker container that will be torn down immediately when the test type completes, we will continuously reclaim resources from each test type while we are running the tests on Public runners.